handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
-# Requests for comments (RFCs)
+# Requests for Comments (RFCs)
```

Comment:
nit: just curious why you chose to capitalize "Comments" here. The [style guide](https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#capitalization) would indicate that it should remain lowercase.

Reply:
```suggestion
# Requests for comments (RFCs)
```

Essentially I was thinking of it as a "name," but don't feel strongly one way or the other. Same with "Tension"—but I think it feels more internally consistent (and with the styleguide) with your suggested changes applies, so doing so!